src/app/components/color-card-item/color-card-item.component.html

```diff
@@ -91,6 +91,8 @@ <h3 class="m-0 font-weight-light">
         <div class="p-4 mx-4 pt-5">
           <div class="input-group">
             <input
+              [ngStyle]="{ display: 'color.loading' ? '' : 'none' }"
```

Comment:
@AndreasGassmann for some reasons this does not work, the intention would be to hide the whole input field and only show the button with its content in the loading state